lib/trino-collect/src/test/java/io/trino/collect/cache/TestEvictableCache.java

```diff
@@ -108,6 +112,29 @@ public void testEvictByWeight()
         assertThat(cache.asMap().values().stream().mapToInt(String::length).sum()).as("values length sum").isLessThanOrEqualTo(20);
     }
 
+    @Test(timeOut = TEST_TIMEOUT_MILLIS)
+    public void testEvictByTime()
+            throws Exception
+    {
+        final int ttl = 200;
+        final int expectedCalls = 10;
+
+        AtomicInteger counter = new AtomicInteger();
+        LoadingCache<String, String> cache = EvictableCacheBuilder.newBuilder()
+                .expireAfterWrite(ttl, TimeUnit.MILLISECONDS)
```

Comment:
code style: we typically import TimeUnit entries statically, see other cases in this class